migrations/applied_migrations/ProvideGoodsName.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+echo "Applying migration ProvideGoodsName"
+
+echo "Adding routes to conf/app.routes"
+
+echo "" >> ../conf/app.routes
+echo "GET        /provideGoodsName                        controllers.ProvideGoodsNameController.onPageLoad(mode: Mode = NormalMode)" >> ../conf/app.routes
+echo "POST       /provideGoodsName                        controllers.ProvideGoodsNameController.onSubmit(mode: Mode = NormalMode)" >> ../conf/app.routes
+
+echo "GET        /changeprovideGoodsName                  controllers.ProvideGoodsNameController.onPageLoad(mode: Mode = CheckMode)" >> ../conf/app.routes
+echo "POST       /changeprovideGoodsName                  controllers.ProvideGoodsNameController.onSubmit(mode: Mode = CheckMode)" >> ../conf/app.routes
```

Comment:
Should we be adding these camel case routes in the migration?

Reply:
In fact we shouldn't. When I ran sbt, I didn't get any feedback, leading to non camel case in the migration!